core/storage/user/gae_models.py

```diff
@@ -57,6 +57,10 @@ class UserSettingsModel(base_models.BaseModel):
         repeated=True,
         indexed=True,
         choices=[lc['code'] for lc in feconf.ALL_LANGUAGE_CODES])
+    # System language preference (for I18N).
+    # TODO(milit): This field should have fixed choices. For now, validation
```

Comment:
We should validate in the backend as well. Can we have a property in feconf that says what languages are currently supported, and refer to that?